Add tofu gtceu

Zoryn4163 opened the pull request:
Adds tofu processing to gtceu machines

Zoryn4163 committed:
add gtceu tofu processing

Zoryn4163 committed:
slicer not mixer

srdr2k3 commented:
I don't think the food items should be in the same file as Sterilized Growth Medium.
Also if Vegan Delight has an item for a bottled soy milk, then there should be bottling & emptying recipes for that too. (If it already isn't in the pack)

Zoryn4163 committed:
move soymilk stuff to Delightful.js and add canner for bottled soymilk

Zoryn4163 commented:
> I don't think the food items should be in the same file as Sterilized Growth Medium. Also if Vegan Delight has an item for a bottled soy milk, then there should be bottling & emptying recipes for that too. (If it already isn't in the pack)

The original `vegandelight:soymilk` was in that file so I just added more there. I moved the lot to Delightful.js

srdr2k3 reviewed (approved):
Thanks!

Zoryn4163 committed:
add un-canning of soymilk bottle (forgot)

Zoryn4163 commented:
if #600 is merged then this PR should be scrapped
